tests/Badge/Service/TextNormalizerTest.php

```diff
@@ -28,8 +28,10 @@ protected function setUp(): void
 
     /**
      * @dataProvider getBadNumberToConvert
+     *
+     * @phpstan-param int|string $input
      */
-    public function testNumberToTextConversion(string $input, string $output): void
+    public function testNumberToTextConversion($input, string $output): void
```

Comment:
in which use case do we receive a int?

Reply:
https://github.com/PUGX/badge-poser/blob/d3f75e4f3a554708bb3515cb32a85ebb588d8939/tests/Badge/Service/TextNormalizerTest.php#L49

Don't know if this scenario is possible, but the goal of this PR was to restore codecoverage, not to "fix" unconsistent tests. If an `int` won't ever be possible in this situation, we should remove it from data provider (another issue/PR from my standpoint).

Reply:
I think that if the use case is present in data provider then that's right as you did! 👍